static/js/popovers.js

```diff
@@ -1032,6 +1180,74 @@ exports.register_click_handlers = function () {
             last_scroll = date;
         });
     }());
+
+    $("body").on("click", "#send_later", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        popovers.toggle_send_later_popover(this);
+    });
+
+    $("body").on("click", ".send_later_hours", function (e) {
+        var send_later_in = $(e.currentTarget).attr('id');
+        var send_at_time = moment().add(send_later_hours[send_later_in].hours, 'h');
+        var message_content = compose_state.message_content();
+        var command = "/schedule " + send_at_time.format("MMM D YYYY h:mm a") + "\n";
+
+        if (reminder.is_deferred_delivery(message_content)) {
+            var raw_message = message_content.split('\n');
+            message_content = raw_message.slice(1).join('\n');
+        }
+
+        message_content = command + message_content;
+        compose_state.message_content(message_content);
+        popovers.hide_send_later_popover();
+    });
+
+    $("body").on("click", ".send_later_tomorrow", function (e) {
+        var send_later_at = $(e.currentTarget).attr('id');
+        var send_time = send_later_tomorrow[send_later_at].time;
+        var date = new Date();
+        var scheduled_date = date.setDate(date.getDate() + 1);
+        var send_at_time = moment(scheduled_date).format("MMM D YYYY ") + send_time;
+        var message_content = compose_state.message_content();
+        var command = "/schedule " + send_at_time + "\n";
+
+        if (reminder.is_deferred_delivery(message_content)) {
+            var raw_message = message_content.split('\n');
+            message_content = raw_message.slice(1).join('\n');
+        }
+
+        message_content = command + message_content;
+        compose_state.message_content(message_content);
+        popovers.hide_send_later_popover();
+    });
+
+    $("body").on("click", ".send_later_days_and_weeks", function (e) {
+        var send_later_in = $(e.currentTarget).attr('id');
+        var send_at_time = moment().add(send_later_days_and_weeks[send_later_in].days, 'd');
+        var message_content = compose_state.message_content();
+        var command = "/schedule " + send_at_time.format("MMM D YYYY h:mm a") + "\n";
+
+        if (reminder.is_deferred_delivery(message_content)) {
+            var raw_message = message_content.split('\n');
+            message_content = raw_message.slice(1).join('\n');
+        }
+
+        message_content = command + message_content;
+        compose_state.message_content(message_content);
+        popovers.hide_send_later_popover();
+    });
+
+    $("body").on("click", "#send-later-custom-input", function (e) {
+        $(e.currentTarget)[0]._flatpickr.toggle();
+        $('.flatpickr-calendar').addClass('send-later-flatpickr');
+        $('.flatpickr-calendar').css('left', '31%');
+        $('.flatpickr-calendar').css('top', "52.8%");
+        $('.flatpickr-calendar').removeClass('arrowBottom');
```

Comment:
Adding a flag here. I could not override styles of third party flatpickr, by simply adding classes and IDs. Adding classes explicitly worked. `flatpickr-calender` class would always override any styles of IDs or classes added to it.

Reply:
OK -- can you add a comment at the start of this section containing that explanation?  I feel like it's something we'll want to have handy for future edits to that code.